src/audio-worklet/migration/worklet-recorder/app.js

```diff
@@ -105,16 +105,18 @@ function handleRecording(processorPort, recordingProperties) {
   const recordButton = document.querySelector('#record');
   const recordText = recordButton.querySelector('span');
   const player = document.querySelector('#player');
-  const downloadButton = document.querySelector('#download');
+  const downloadLink = document.querySelector('#download-link');
+  const downloadButton = document.querySelector('#download-button');
 
   let recordingLength = 0;
 
   // If the max length is reached, we can no longer record.
   const recordingEventCallback = async (event) => {
     if (event.data.message === 'MAX_RECORDING_LENGTH_REACHED') {
       isRecording = false;
-      recordText.innerHTML = 'Start';
-      recordButton.setAttribute.disabled = true;
+      recordText.innerHTML = 'Ready to download 5 mins';
```

Comment:
Why "5 mins"? The recorded audio might not be 5-minute long.

Reply:
It is because this is the time where main thread receive the 'MAX_RECORDING_LENGTH_REACHED' message from processor. We know that the total length of recording must be 5 mins and we don't want the recording button functional anymore, so I disabled it. 

If we stop recording before 5 mins, we still want the record button functional to resume the recording.